owiddata/01.OWID.basicStats.py

```diff
@@ -70,13 +71,29 @@ def main(args):
     # Transform list of vaccine candidate per iso code to list of ISO codes per vaccine candidate
     allVaxByCountry = dict(zip(vaccine_locations["iso_code"],
                                vaccine_locations["vaccines"]))
+    owid_stats["owid_total_countries"] = str(len(allVaxByCountry))
+
     countryByVax = dict()
     for iso, vaccines in allVaxByCountry.items():
         for vax in vaccines.split(","):
             vax = vax.strip()
             countryCodes = countryByVax.get(vax, [])
             countryByVax[vax] = countryCodes + [iso]
 
+    # Pull list of countries where specific vaccines have been authorized, as requested by authors
+    for vaccine in ["Covaxin","Sinovac"]:
+        isos = countryByVax[vaccine]
+        owid_stats[vaccine+"_country_count"] = str(len(isos))
+
+        country_data = setup_geopandas()
+        continents_df = (
+                country_data[country_data['iso_a3'].isin(isos)]
+                >> ply.select("continent")
+                >> ply.distinct(["continent"])
+        )
+        continents = [value for row in continents_df.values.tolist() for value in row]
+        owid_stats[vaccine + "_continents"] = ', '.join(continents[:-1]) + " and " + continents[-1]
```

Comment:
```suggestion
        owid_stats[vaccine + "_continents"] = ', '.join(continents[:-1]) + ", and " + continents[-1]
```
Should we sort the list of continents? It seems fairly arbitrary now.

Reply:
I'm also adding the Oxford comma!